app/policies/case_contact_policy.rb

```diff
@@ -1,15 +1,14 @@
 class CaseContactPolicy < ApplicationPolicy
   def new?
-    # Everyone is allowed to create a case_contact
     true
   end
 
   def _is_creator_or_casa_admin?
-    _is_admin? || _is_creator?
+    is_admin? || _is_creator?
   end
 
   def _is_creator_or_supervisor_or_casa_admin?
```

Comment:
Not that you should fix this, but this doesn't look like idiomatic ruby to me. Am I missing something that the `_method_name` implies?

Reply:
@noahc I'm pretty sure I removed all of those. I think it's a python naming convention to indicate a private method.

Reply:
I went commit by commit. I saw you did that later on.